src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -18,6 +18,20 @@ namespace Stratis.Bitcoin.Features.BlockStore
     /// </summary>
     public interface IBlockRepository : IBlockStore
     {
+        /// <summary> The dbreeze database engine.</summary>
+        DBreezeEngine DBreeze { get; }
+
+        /// <summary>
+        /// Delete blocks and their transactions.
```

Comment:
deletes blocks and indexes for transactions that belong to deleted blocks

Reply:
Fixed